guides/source/active_storage_overview.md

```diff
@@ -52,20 +52,26 @@ below declares three services named `local`, `test`, and `amazon`:
 ```yaml
 local:
   service: Disk
+  environment: development
   root: <%= Rails.root.join("storage") %>
 
 test:
   service: Disk
+  environment: test
   root: <%= Rails.root.join("tmp/storage") %>
 
 amazon:
   service: S3
+  environment: production
   access_key_id: ""
   secret_access_key: ""
   bucket: ""
   region: "" # e.g. 'us-east-1'
 ```
 
+The `environment` configuration is optional, but makes Active Storage more secure
+by preventing access to the specified except when Rails is running in the specified environment.
```

Comment:
"by preventing access to the specified environment except when Rails is running in said environment."

Reply:
```suggestion
by preventing access to the specified service except when Rails is running in the stated environment.
```